src/R/Components/Impl/PackageManager/Implementation/View/PackageDetails.xaml.cs

```diff
@@ -14,15 +17,23 @@ public partial class PackageDetails : UserControl {
         }
 
         private void HyperlinkUrl_Click(object sender, RoutedEventArgs e) {
-            throw new System.NotImplementedException();
+            throw new NotImplementedException();
         }
 
         private void ButtonInstall_Click(object sender, RoutedEventArgs e) {
-            throw new System.NotImplementedException();
+            throw new NotImplementedException();
         }
         
         private void ButtonUninstall_Click(object sender, RoutedEventArgs e) {
-            throw new System.NotImplementedException();
+            throw new NotImplementedException();
+        }
+
+        private void RepositoryUri_RequestNavigate(object sender, RequestNavigateEventArgs e) {
+            Process.Start(e.Uri.AbsoluteUri);
+        }
+
+        private void LibraryPath_RequestNavigate(object sender, RequestNavigateEventArgs e) {
+            Process.Start(e.Uri.ToString().Replace('/', '\\'));
```

Comment:
It's a path on disk in the following format:
```
> i1[1,'LibPath']
[1] "C:/Users/huvalo/Documents/R/win-library/3.2"
> 
```

Also, I've never seen it return a path with a ` in it, it's always expanded.  So the code above should work AFAIK.